src/utils/dataTestIds.ts

```diff
@@ -428,4 +428,7 @@ export const dataTestId = {
     newRegistrationLink: 'new-registration-link',
     researchProfileLink: 'research-profile-link',
   },
+  tasksPage: {
+    approvalAccordion: 'approval-accordion',
```

Comment:
bare ta dette også med en gang?
```suggestion
    userDialogAccordion: 'user-dialog-accordion',
```